src/elektra/redshift.ini

```diff
@@ -0,0 +1,234 @@
+; Elektra specification file for redshift@490ba2aae9cfee097a88b6e2be98aeb1ce990050 (= redshift v1.12 + some small commits)
+; This file is based on: https://github.com/ElektraInitiative/libelektra/blob/master/examples/spec/lcdproc/LCDd.ini
```

Comment:
There is barely anything in common anymore, I do not think this statement is relevant.

Reply:
Good point, I removed it